docs/install/configuration.md

```diff
@@ -37,13 +38,54 @@ token = xyz
 
 ### Locations
 
-Locations are validated against a set of known location names and possible
-aliases for those names. The data resource providing this information is set
-with the `location` configuration, providing to local JSON formatted file containing
-known locations and location aliases. An example of this file format can be
-downloaded from:
+Locations are validated against a set of known location names and possible aliases for
+those names. The data resources providing this information are set with the `gazetteer`
+and `location_aliases` configuration options.
+
+The gazetteer file must be a [GeoJSON](https://geojson.org/) file containing a
+collection of GIS features providing known locations for a project. The GIS features can
+be simple point locations but the file can also include linestrings or polygons to
+capture linear features like streams or transects and area features like quadrats. Each
+feature will have a set of properties: `safedata_validator` only requires the `location`
+property but it is fine for features to have other project specific properties. The
+values of `location` **must be unique** within the gazetteer file.
+
+A simple minimal example of GeoJSON file content is shown below, but this file format is
+commonly used in GIS applications, so can be created and edited using GIS tools.
+
+```json
+{    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+                "location": "location_name"
+            },
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    117.586071,
+                    4.710346
+                ]
+            }
+        }
+    ]
+}
+```
+
+The location aliases file provides a way to handle commonly used alternative names for
+sampling locations. The file format is a simple CSV file which must contain the field
+header shown below:
+
+```csv
+location,alias,zenodo_record_id
```

Comment:
Is it worth mentioning why a zenodo record id is an option here?

Reply:
Yeah - simple answer is that I couldn't fully remember and was parking it for now 😞 However, that is now being worked on as I fix parsing these at the server side so will update.

Reply:
Ahh yeah fair enough to park for now

Reply:
I've updated with comments